packages/common/bucketsConfigJsonObject.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const aws = require('./aws');
+/**
+ * Retrieve the stack's bucket configuration from s3 and return the bucket configuration object.
+ *
+ * @param {string} bucket - system bucket name.
+ * @param {string} stackName - stack name.
+ * @returns {Object} - stack's bucket configuration.
+ */
+async function bucketsConfigJsonObject(
```

Comment:
Should we unit test this by staging a JSON file in localstack?

Reply:
It seems a little testing AWS, but sure.

Reply:
c4c5aed43